src/frontend/src/Components/shared/Filters.scss

```diff
@@ -41,6 +41,42 @@ button.btn.open-filters {
   .filters-title {
     margin: 1rem 1rem 0;
     font-size: 1.125rem;
+    margin-right: 0rem;
+    &__container {
+      display: flex;
+      align-items: center; 
+      justify-content: space-between; 
+      gap: 0.1rem; 
+    }
+
+    &__notification {
+      font-size:0.875rem;
+      color:$White; 
+      background-color: $Blue100; 
+      padding: 0.2rem; 
+      border-radius: 4px; 
+      display: inline-block; 
+      flex-shrink: 1; 
+      margin-right: 3rem;
+      margin-top: 0.875rem;
+    }
```

Comment:
please update the following:
- font-size: 0.75rem;
- background-color: $Focus;
- padding: 0.25rem 0.375rem;

add:
- margin-left: 8px;

remove:
- display: inline-block;
- flex-shrink: 1;